fastkml/kml.py

```
@@ -208,6 +210,53 @@
+        Write KML to a file
+        Args:
+        ----
+            file_name: The file name where to save the file.
+                Can be any string value
+            prettyprint : bool, default=True
+                Whether to pretty print the XML.
+            xml_declarationL bool, default=True
+                For True, value is '<?xml version="1.0" encoding="UTF-8"?>' else ''
+
+        """
+        element = self.etree_element()
```

Ruff

First line should end with a period

Add period

(D400)